contracts/Minter.sol

```diff
@@ -1,35 +1,41 @@
-// SPDX-License-Identifier: GPL-3.0-or-later
+// SPDX-License-Identifier: MIT
 pragma solidity 0.8.13;
 
-import "contracts/libraries/Math.sol";
+import "openzeppelin-contracts/contracts/utils/math/Math.sol";
+
 import "contracts/interfaces/IMinter.sol";
 import "contracts/interfaces/IRewardsDistributor.sol";
-import "contracts/interfaces/IVelo.sol";
+import "contracts/interfaces/IFlow.sol";
 import "contracts/interfaces/IVoter.sol";
 import "contracts/interfaces/IVotingEscrow.sol";
 
 // codifies the minting rules as per ve(3,3), abstracted from the token to support any token that allows minting
 
 contract Minter is IMinter {
-    uint internal constant WEEK = 86400 * 7; // allows minting once per week (reset every Thursday 00:00 UTC)
-    uint internal constant EMISSION = 990;
-    uint internal constant TAIL_EMISSION = 2;
-    uint internal constant PRECISION = 1000;
-    IVelo public immutable _velo;
+    uint256 internal constant WEEK = 86400 * 7; // allows minting once per week (reset every Thursday 00:00 UTC)
+    uint256 internal constant EMISSION = 990;
+    uint256 internal constant TAIL_EMISSION = 2;
+    uint256 internal constant PRECISION = 1000;
+    IFlow public immutable _flow;
     IVoter public immutable _voter;
     IVotingEscrow public immutable _ve;
     IRewardsDistributor public immutable _rewards_distributor;
-    uint public weekly = 15_000_000 * 1e18; // represents a starting weekly emission of 15M VELO (VELO has 18 decimals)
-    uint public active_period;
-    uint internal constant LOCK = 86400 * 7 * 52 * 4;
+    uint256 public weekly = 15_000_000 * 1e18; // represents a starting weekly emission of 15M FLOW (FLOW has 18 decimals)
+    uint256 public active_period;
+    uint256 internal constant LOCK = 86400 * 7 * 52 * 4;
 
     address internal initializer;
     address public team;
     address public pendingTeam;
-    uint public teamRate;
-    uint public constant MAX_TEAM_RATE = 50; // 50 bps = 0.05%
+    uint256 public teamRate;
+    uint256 public constant MAX_TEAM_RATE = 50; // 5% max
```

Comment:
[Question] Just want to make sure we want 0.5% max team rate?

Reply:
yep